basic/src/test/java/org/jboss/hal/testsuite/test/configuration/connector/ResourceAdaptersTestCase.java

```diff
@@ -115,10 +115,9 @@ public void createProperties() {
         ConfigPropertiesFragment fragment = area.switchToProperty();
         ConfigPropertyWizard wizard = fragment.addProperty();
         Console.withBrowser(browser).waitUntilLoaded();
-
+        wizard.getEditor().text("name", PROPERTY_KEY);
```

Comment:
This is sudden inconsistency in property wizard dialog? If this is supposed to be different type of wizard, we should create abstraction for it, if not, we should make sure that there is consistent approach.